jax/_src/lax/linalg.py

```diff
@@ -1042,7 +1042,7 @@ def body(k, state):
     a = a.at[:, k].set(jnp.where(m_idx > k, a[:, k] / x, a[:, k]))
 
     # a[k+1:, k+1:] -= jnp.outer(a[k+1:, k], a[k, k+1:])
-    a = a - jnp.where((m_idx[:, None] > k) & (n_idx > k),
+    a = a - jnp.where((m_idx[:, None] > k) & (n_idx[:, None] > k),
```

Comment:
I think this changes the semantics of the operation. I think it should be `n_idx[None, :]`

Reply:
Yes, you're correct.
That's what I get for not rerunning the tests before putting up the PR. 😅